web/docs/integrations/google.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Well this is awesome, proper instructions!! This is it then, full details how to open create Google app right?

Reply:
Thanks, I created this by doing the full walkthrough myself and using the generated client id/secret locally to make sure it all worked. This should be all they need to get going. 👍🏻